### PR TITLE
Feature/mf27

### PR DESCRIPTION
**Pull request by whaeck**
MF27 is now added. Like MF23 it is quite simple and almost identical to MF3.

This builds on #138 so that one needs to be merged first.

### CODE REVIEW

File: src/ENDFtk/section/27.hpp [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+    /**
+     *  @brief Return the energy values
+     */
+    auto energies() const { return this->table.x(); }
+
+    /**
+     *  @brief Return the form factor or scattering function values
+     */
+    auto values() const { return this->table.y(); }
```
Comment (nathangibson14) [inline src/ENDFtk/section/27.hpp:72]: Should you include the ENDF-speak of `H` for this?

**Review by nathangibson14 — CHANGES_REQUESTED** [review]
Because MF27 data can mean different things, I'm a little iffy on the interface.  I think I'm okay with `energies` being something that can be returned, but I'd definitely change the constructor to be the more general `x`.

That all assumes that x isn't necessarily an energy.  The equations in the ENDF manual don't really make sense, as x appears without definition, and I don't know the physics enough to intuit it.  Do you know any more?

File: src/ENDFtk/section/27.hpp [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+    /**
+     *  @brief Return the interpolation boundaries
+     */
+    auto boundaries() const { return this->table.boundaries(); }
+
+    /**
+     *  @brief Return the energy values
+     */
+    auto energies() const { return this->table.x(); }
```
[the remark this answers is not in the document]
Comment (whaeck) [reply src/ENDFtk/section/27.hpp]: I've replaced this with X and H, no human readable version.

File: src/ENDFtk.hpp [hunk 3]
```diff
@@ -19,8 +19,14 @@
...
 #include "ENDFtk/section/7.hpp"
 #include "ENDFtk/section/8.hpp"
+#include "ENDFtk/section/9.hpp"
+#include "ENDFtk/section/10.hpp"
 #include "ENDFtk/section/12.hpp"
 #include "ENDFtk/section/13.hpp"
+#include "ENDFtk/section/14.hpp"
+#include "ENDFtk/section/15.hpp"
+#include "ENDFtk/section/23.hpp"
+#include "ENDFtk/section/27.hpp"
```
Comment (nathangibson14) [inline src/ENDFtk.hpp:29]: I don't know if this is my fault or yours, but this is missing 33.hpp

File: src/ENDFtk.hpp [hunk 4]
```diff
@@ -31,8 +37,14 @@
...
 #include "ENDFtk/file/7.hpp"
 #include "ENDFtk/file/8.hpp"
+#include "ENDFtk/file/9.hpp"
+#include "ENDFtk/file/10.hpp"
 #include "ENDFtk/file/12.hpp"
 #include "ENDFtk/file/13.hpp"
+#include "ENDFtk/file/14.hpp"
+#include "ENDFtk/file/15.hpp"
+#include "ENDFtk/file/23.hpp"
+#include "ENDFtk/file/27.hpp"
```
Comment (nathangibson14) [inline src/ENDFtk.hpp:47]: Also missing 33.

**Review by nathangibson14 — CHANGES_REQUESTED** [review]
One last fix needed, I'll just do it myself then approve.

**Review by nathangibson14 — APPROVED** [review]

**Pull request merged**